src/apps/api/views/mixins.py

```diff
@@ -0,0 +1,40 @@
+# NOTE: We don't have delete mixin
+from rest_framework import status
+from rest_framework.response import Response
+
+
+class BulkViewSetMixin(object):
```

Comment:
No. So Python mixins generally always use `object` as a base. Because of inheritance this will override the defined methods/fields on whatever secondary class we're inheriting from. (In this use case we're generally using ModelViewSet as the secondary class. (I could be backwards on the order...))